src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +30,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```

Comment:
This needed to happen, but maybe method indicate that it may throw this exception?